src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
I'm of two minds with strength and stamina loss from melee.  On one hand, it makes sense that stronger characters aren't exerting as much.  On the other hand, they're already getting more damage per swing, doesn't that mean they're exerting themselves more?